sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -1070,8 +1163,19 @@ Page<string> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
+        // TODODOCS.
         /// <summary>
+        /// Gets data about the data points present in the specified set of time series.
+        /// <see cref="MetricAnomalyAlertConfiguration"/>.
         /// </summary>
+        /// <param name="seriesKeys">The name of the dimension.</param>
+        /// <param name="detectionConfigurationId">The unique identifier of the <see cref="MetricAnomalyAlertConfiguration"/>.</param>
+        /// <param name="startTime">Filters the result. Only data points ingested after this point in time, in UTC, will be returned.</param>
+        /// <param name="endTime">Filters the result. Only data points ingested after this point in time, in UTC, will be returned.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A collection containing the values the specified dimension assumed for anomalous data points. Items are unique.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="seriesKeys"/> or <paramref name="detectionConfigurationId"/> is null.</exception>
+        /// <exception cref="ArgumentException"><paramref name="seriesKeys"/> or <paramref name="detectionConfigurationId"/> is empty; or <paramref name="detectionConfigurationId"/> is not a valid GUID.</exception>
```

Comment:
This one still needs clarification. You may ignore this whole block, as it will be updated in the next batch.